package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/tinymce-react",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
Any reason you're bumping the version here as we're only updating the dev dependencies? Also if you are releasing a new version, then we also need to update the changelog.

- [ ] Changelog or leave this as is for now

Reply:
Ok, I thought we had to do a release. I will remove the version bumps.

Reply:
Yeah, I should have clarified that earlier sorry!

Reply:
Can you please release a new version?

Reply:
@tinydylan what's the purpose? Just to pull in the fix done the other day for the `utils` maybe?

Reply:
As discussed out-of-band, since this resolves a security alert, we should release an update.

Reply:
I wasn't included in the discussion and it was my concern, so please include me next time :unamused: